services/user.py

```diff
@@ -0,0 +1,52 @@
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str | None = None,
+                first_name: str | None = None,
+                last_name: str | None = None) -> None:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency).